src/styles/components/_c-landing.scss

```diff
@@ -19,41 +26,39 @@
       bottom: 0;
     }
     .phone-image {
-      left: 0;
-      height: 788px;
-      background-size: contain;
       background-image: url(/images/phone@2x.png) !important;
```

Comment:
Unless you're dealing with styles set by a third-party library and there's no other resort, avoid using `!important`. In this case I think you can remove it and nothing will change.